subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/encoding/Identifier.java

```diff
@@ -27,6 +27,7 @@
     private static final String NON_PRECOMPOSED_NON_ASCII = "-√æ∫ʙ₦∆√∫";
     private static final String FILESYSTEM_RESERVED_CHARS = "-./\\?%*:|\"<>";
     private static final String XML_MARKUP_CHARS = "-<with>some<xml-markup/></with>";
+    private static final String GRADLE_NAME_FORBIDDEN_CHARACTERS = " /\\:"; //See: NameValidator.FORBIDDEN_CHARACTERS
```

Comment:
That does not work as. NameValidator is not on the classpath here (`subprojects/internal-testing`). Otherwise I would have used `NameValidator.FORBIDDEN_CHARACTERS` directly here.